apps/src/templates/teacherDashboard/ParentLetter.jsx

```diff
@@ -107,24 +110,41 @@ class ParentLetter extends React.Component {
           <h1>{i18n.parentLetterStep2()}</h1>
           <SignInInstructions
             loginType={loginType}
+            loginTypeName={loginTypeName}
             secretPicturePath={secretPicturePath}
             secretWords={secretWords}
             sectionCode={sectionCode}
             studentName={studentName}
           />
-          <SafeMarkdown
-            markdown={i18n.parentLetterStep2Details({
-              studentName: studentName,
-              projectsLink: studio('/projects/public'),
-              atHomeLink: pegasus('/athome'),
-            })}
-          />
-          <h1>{i18n.parentLetterStep3()}</h1>
-          <SafeMarkdown
-            markdown={i18n.parentLetterStep3Details({
-              accountEditLink: studio('/users/edit'),
-            })}
-          />
+          {loginType !== SectionLoginType.lti_v1 ? (
+            <SafeMarkdown
+              markdown={i18n.parentLetterStep2Details({
+                studentName: studentName,
+                projectsLink: studio('/projects/public'),
+                atHomeLink: pegasus('/athome'),
+              })}
+            />
+          ) : (
+            <SafeMarkdown
+              markdown={i18n.parentLetterStep2Details_LMS({
+                studentName: studentName,
+                loginTypeName: loginTypeName,
+              })}
+            />
+          )}
+          {loginType !== SectionLoginType.lti_v1 ? (
+            <div>
+              <h1>{i18n.parentLetterStep3()}</h1>
+              <SafeMarkdown
+                markdown={i18n.parentLetterStep3Details({
+                  accountEditLink: studio('/users/edit'),
+                })}
+              />
+            </div>
+          ) : (
+            <div />
+          )}
+
```

Comment:
To avoid the extra ternary operators, would it be possible to have two different functions `getLtiInstructions` and `getRegularInstructions` and we render the appropriate instructions based on the login type?